.circleci/scripts/vs_install.ps1

```diff
@@ -13,10 +16,25 @@ $VS_INSTALL_ARGS = @("--nocache","--quiet","--wait", "--add Microsoft.VisualStud
 
 curl.exe --retry 3 -kL $VS_DOWNLOAD_LINK --output vs_installer.exe
 if ($LASTEXITCODE -ne 0) {
-    echo "Download of the VS 2017 installer failed"
+    echo "Download of the VS 2019 Version 16.8.5 installer failed"
     exit 1
 }
 
+if (Test-Path "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe") {
+    $existingPath = & "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -products "Microsoft.VisualStudio.Product.BuildTools" -version "[16, 17)" -property installationPath
```

Comment:
-version "[16,17]"
Is this necessary?

Reply:
It is used to limit `vswhere` to find vs 2019 only.